packages/earn-controller/src/EarnController.ts

```diff
@@ -377,30 +508,38 @@ export class EarnController extends BaseController<
    * @param options - Optional arguments
    * @param [options.resetCache] - Control whether the BE cache should be invalidated (optional).
    * @param [options.address] - The address to refresh pooled stakes for (optional).
+   * @param [options.chainId] - The chain id to refresh pooled stakes for (optional).
    * @returns A promise that resolves when the stakes data has been updated
    */
   async refreshPooledStakes({
     resetCache = false,
     address,
+    chainId,
```

Comment:
Perhaps we ought to document that `refreshPooledStakes` now takes an optional `chainId`, and it can be used to control which chain pooled stakes will be fetched for?